llpc/translator/lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -1555,13 +1556,29 @@ class SPIRVExtInst : public SPIRVFunctionCallGeneric<OpExtInst, 5> {
       getDecoder(I) >> ExtOpNonSemanticInfo;
       break;
     case SPIRVEIS_Debug:
+    case SPIRVEIS_NonSemanticShaderDebugInfo100:
       getDecoder(I) >> ExtOpDebug;
       break;
     default:
       assert(0 && "not supported");
       getDecoder(I) >> ExtOp;
     }
     getDecoder(I) >> Args;
+
+    if (ExtSetKind == SPIRVEIS_NonSemanticShaderDebugInfo100) {
```

Comment:
This gives me a warning on clang:
```
llpc/translator/lib/SPIRV/libSPIRV/SPIRVInstruction.h:1788:22: warning: comparison of different enumeration types ('SPIRV::SPIRVDebugExtOpKind' (aka 'SPIRVDebug::Instruction') and 'NonSemanticShaderDebugInfo100Instructions') [-Wenum-compare]
```

Reply:
this PR https://github.com/GPUOpen-Drivers/llpc/pull/2385/files ?